core/templates/dev/head/components/SubtitledHtmlEditorDirective.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the subtitled html editor.
+ */
+
+oppia.directive('subtitledhintEditor', [
```

Comment:
It is used for purposes other than just hint. I think `subtitleHtmlEditor` would be a better name

Reply:
okay